src/Watchy.cpp

```diff
@@ -114,6 +114,7 @@ void Watchy::handleButtonPress() {
     } else if (guiState == FW_UPDATE_STATE) {
       showMenu(menuIndex, false); // exit to menu if already in app
     } else if (guiState == WATCHFACE_STATE) {
+      button1();
```

Comment:
How about:

```suggestion
      handleButton2Press();
```

This lines up with the function naming pattern that already exists, `handleButtonPress`.

Also, this is SW2 on the PCB, and here is SW2 documented in the Getting Started guide: https://watchy.sqfmi.com/docs/getting-started/

![image](https://user-images.githubusercontent.com/820984/168224518-b0495793-f482-4981-b217-dfbf46934f91.png)